com.unity.ml-agents.extensions/Runtime/QTTransform.cs

```diff
@@ -0,0 +1,96 @@
+using UnityEngine;
+
+namespace Unity.MLAgents.Extensions
+{
+    /// <summary>
+    /// Simple linear transform representation consisting of a rotation and translation.
+    /// When deriving the math, it can be helpful to think of the transform as a 4x4 block matrix:
+    /// <example>
+    /// | R | t |
+    /// ----+----
+    /// | 0 | 1 |
+    /// </example>
+    /// where R is a 3x3 rotation, t is a 3x1 translation, 0 is a 1x3 vector of 0s.
+    /// </summary>
+    public struct QTTransform
```

Comment:
QTTransform = Quaternion + Translation Transform.

Reply:
I was just informed of the existence of https://docs.unity3d.com/ScriptReference/Pose.html - I think that might be usable instead (although might need to add an inverse extension method)